real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/DataPanel.java

```diff
@@ -136,6 +137,10 @@ public void actionPerformed(String topicName, Message m) {
             }
         });
 
+        new Subscriber("compass", NodeChannel.IMU_COMPASS.getMsgPath(), ((topicName, m) -> {
```

Comment:
This was meant for displaying the value during testing, I'll take it out and submit another PR later for gui mods